hyperkit.1

```diff
@@ -356,6 +356,24 @@ hyperkit -c 4 \\
 
 .Ed
 
+.Sh EXIT STATUS
+The
+.Nm hyperkit
+process exits with one of the following values:
+.Pp
+.Bl -tag -width flag -compact
+.It Li 0
+The guest either powered down or halted cleanly.
+.It Li 1
+An error occurred.
+.It Li 2
+The guest rebooted/reset. The process may needs restarting.
```

Comment:
`s/needs/need/`.

I think there is also no "may" about it -- if the caller wants reboot to mean something then it must restart. How about "The guest rebooted/reset. The caller is responsible for starting a new hyperkit process in this case"?